src/app/modules/form/admin-form/admin-form.utils.ts

```diff
@@ -586,3 +588,36 @@ export const mapGoGovErrors = (error: AxiosError): GoGovError => {
       )
   }
 }
+
+// TODO: remove once we upgrade to Mongoose 7.x
+/**
+ * Manually insert default validation options for short text columns in tables.
+ * Due to a bug in Mongoose 6.x, default values are not inherited for discriminators.
+ *
+ * See https://github.com/Automattic/mongoose/issues/12135
+ *
+ * Fixed in Mongoose 7.x
+ * @param newField
+ * @returns
+ */
+
+export const insertTableShortTextColumnDefaultValidationOptions = (
+  newField: FieldUpdateDto,
+) => {
+  if (newField.fieldType === BasicField.Table) {
+    const defaultValidationOptions = {
```

Comment:
```suggestion
    const defaultValidationOptions: TextValidationOptions = {
```
(import TextValidationOptions from '../../../../../shared/types')

Reply:
Makes sense. Added in b8cb4665